### PR TITLE
XSIAM | ForcepointSWG Pack

### PR DESCRIPTION
**Pull request by nkanon**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
This PR is created to add a pack for Forcepoint Web Sense Gateway

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0
- [x] 8.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/ForcepointSWG/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+
+# Forcepoint Secure Web Gateway
+This pack includes Cortex XSIAM content.
+
+
+## Configuration on Server Side
+Use the **Settings** > **General** > **SIEM Integration** page to configure Websense software to send log data from Filtering Service to a supported Security Information and Event Management (SIEM) solution.
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointSWG/README.md:7]:
```suggestion
In the **Settings** > **General** > **SIEM Integration** page you can configure Websense software to send log data from the Filtering Service to a supported Security Information and Event Management (SIEM) solution.
```

File: Packs/ForcepointSWG/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Before using this page to enable SIEM integration, make sure an instance of Websense Multiplexer is installed for each Policy Server in your deployment.
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointSWG/README.md:8]:
```suggestion
Before enabling the SIEM integration, make sure an instance of Websense Multiplexer is installed for each Policy Server in your deployment.
```

File: Packs/ForcepointSWG/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+Perform these steps for each Policy Server instance in your deployment:
+1. Select Enable SIEM integration for this Policy Server to turn on the SIEM integration feature.
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointSWG/README.md:11]:
```suggestion
1. Select **Enable SIEM integration for this Policy Server** to turn on the SIEM integration feature.
```

File: Packs/ForcepointSWG/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+2. Provide the IP address of the BrokerVM, as well as the communication Port to use for sending the data.
+3. Specify the Transport protocol (UDP or TCP) to use when sending data to XSIAM.
+4. Select **syslog/CEF** format to use. This determines the syntax of the string used to pass log data to the integration.
+5. Click OK to cache your changes. Changes are not implemented until you click Save and Deploy.
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointSWG/README.md:15]:
```suggestion
5. Click **OK** to cache your changes. Changes are not implemented until you click **Save and Deploy**.
```

File: Packs/ForcepointSWG/README.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+When you save your changes, Websense Multiplexer connects to Filtering Service and takes over the job of distributing log data to both Log Server and the selected SIEM integration.
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointSWG/README.md:17]:
```suggestion
When you save your changes, Websense Multiplexer connects to the Filtering Service and takes over the job of distributing log data to both the Log Server and the selected SIEM integration.
```

File: Packs/ForcepointSWG/README.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+In order to use the collector, use the [Broker VM](#broker-vm) option.
+
+
+### Broker VM
+To create or configure the Broker VM, use the information described [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/Cortex-XDR-Pro-Administrator-Guide/Configure-the-Broker-VM).
+
+You can configure the specific vendor and product for this instance.
+
+1. Navigate to **Settings** > **Configuration** > **Data Broker** > **Broker VMs**. 
+2. Go to the apps tab and add the **Syslog** app. If it already exists, click the **Syslog** app and then click **Configure**.
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointSWG/README.md:29]:
```suggestion
2. Click the **Apps** tab and add the **Syslog** app. If it already exists, click the **Syslog** app and then click **Configure**.
```

File: Packs/ForcepointSWG/README.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+3. Click **Add New**.
+4. When configuring the Syslog Collector, set the following values:
```
Comment (ShirleyDenkberg) [inline Packs/ForcepointSWG/README.md:31]:
```suggestion
4. When configuring the Syslog Collector, set the following value:
```

**Comment by ShirleyDenkberg** [comment]
@nkanon Doc review completed.

**Comment by nkanon** [comment]
> @nkanon Doc review completed.
Thanks @ShirleyDenkberg !

**Review by evisochek — APPROVED** [review]

**Pull request merged**